app/models/custom_attribute.rb

```diff
@@ -2,7 +2,17 @@ class CustomAttribute < ApplicationRecord
   belongs_to :resource, :polymorphic => true
   serialize :serialized_value
 
+  def value=(value)
```

Comment:
We might decide to remove it later on, you can see: https://github.com/ManageIQ/manageiq/pull/10896#issuecomment-245008657
But how come you think its leaking into the public API?